src/guides/creatingbotaccount.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
would be possible to add maybe some screenshots? if that wouldn't that much of hassle

Reply:
sure - is there a preferred way to do that? i.e. image provider vs uploading to the repo

Reply:
In repo would be more stable i guess. And should be achived pretty easily by adding to base